rllib/algorithms/algorithm.py

```diff
@@ -1363,6 +1366,38 @@ def _evaluate_with_custom_eval_function(self) -> Tuple[ResultDict, int, int]:
 
         return eval_results, env_steps, agent_steps
 
+    def _evaluate_offline_on_local_runner(self):
+        # if hasattr(env_runner, "input_reader") and env_runner.input_reader is None:
```

Comment:
remove this comment?

Reply:
Oh yeah! How did this even get in there?